### DISCUSSION TITLE
Epics

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#52390] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It would be great if I could define epics and assign issues/tasks to epics

**Comment by QuintinWillison** [comment]
Agreed. This is a major barrier for adoption of GitHub as "one platform to rule it all" when working on any project that's grown to a non-trivial size, involving even just a handful of developers and managers. We need the ability to be able to nest issues and create hierarchical structure. The current GitHub Projects interface feels somewhat Fisher Price compared to [GitLab's Epics offering](https://docs.gitlab.com/ee/user/group/epics/) or a 'real' tool like Jira.

I did just [find this](https://github.com/orgs/community/discussions/7267#discussioncomment-4638597) so perhaps that'll help fill the void. 🤔

**Reply by QuintinWillison** [reply]
As a quick update in terms of our workaround, until GitHub can give us something better, we've created a field named `Epic` of type `Single select` with short, pithy names for the epics we want to group by in our road map view. Feels a bit clunky but does present a more palatable view for those in upper management.

**Comment by luongvo** [comment]
Really need this feature to be added to GitHub Project soon 🤟

**Comment by douglaslassance** [comment]
Talking from a game development perspective, I would find it important to allow issues to be in multiple epics. I may want to see the status of a specific character or the status of animation in general. If labels had statuses, they'd be pretty close to what I'm envisioning for epics.

Potentially, these epics could be parented to each other in such a way that I could see the status of all characters in my project as well as the status of a specific one.

All that said, it appears that [tasklists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists) is going to be GitHub's way to avoid needing epics. Curious to see if that solves things for me.

**Comment by einaren** [comment]
Ref https://github.com/github/roadmap/issues/837.. Please consider carefully. To me, one of the best features of GitHub Issues is that there are no "Issue Types"! It lets us focus on content instead of endless discussions on which type of Issue some requested change is. Less is more :)

**Comment by Zerotask** [comment]
Besides the previously mentioned [Issue Types](https://github.com/github/roadmap/issues/837) which are planned for Q2 2024, I also saw in the public roadmap, that [Sub-issues](https://github.com/github/roadmap/issues/927) are also planned for Q2 2024. They will replace the task list feature. So there won't be a specific epic feature, but the flexibility to either create issue types like `Epic` and `User Story` and/or use sub-issues
